<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassMembers</class>
 <widget class="QWidget" name="ClassMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="teacher_info" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#d0a8a9;</string>
   </property>
   <widget class="QLabel" name="teacher_name_">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>XXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="teacher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border-radius: 20px;
	border: 5px solid rgb(205, 16, 118);
	background-position: center;</string>
    </property>
    <property name="text">
     <string> 教师</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="student_info" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#d0a8a9;</string>
   </property>
   <widget class="QLabel" name="student_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>XXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="student">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border-radius: 20px;
	border: 5px solid rgb(205, 16, 118);
	background-position: center;</string>
    </property>
    <property name="text">
     <string> 学生</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/ui.qrc"/>
 </resources>
 <connections/>
</ui>
